### PR TITLE
Michal

### PR DESCRIPTION
**Pull request by michalz92**

### CODE REVIEW

File: grizly/orchestrate.py [hunk 1]
```diff
@@ -139,8 +140,8 @@ def update_json(self):
         with open(r"C:\Users\te393828\acoe_projects\infrastructure\listener_store.json", "w") as f_write:
```
Comment (malinaa96) [inline grizly/orchestrate.py]: hey, what path is this?

Comment (michalz92) [reply grizly/orchestrate.py]: Good catch. :D Will fix.

Comment (michalz92) [reply grizly/orchestrate.py]: Done

**Comment by acivitillo** [comment]
Also why are you pushing to master? We should be pushing to 0.2, or?

**Pull request merged**